test/TorchSharpTest/TorchSharpTest.csproj

```diff
@@ -21,6 +21,7 @@
     <None Remove="adamw1.dat" />
     <None Remove="asgd1.dat" />
     <None Remove="bug510.dat" />
+    <None Remove="pyload_test.model" />
```

Comment:
It also needs to be added to the CUDA version of the test project (by link, not copy).

Reply:
Thanks! Will do!